iqmotion/clients/client_files/brushless_drive.json

```diff
@@ -1,35 +1,61 @@
+  {"type_idn":50, "param":"permute_wires",                  "param_idn":  16, "format":"B", "unit": "bool"        },
+  {"type_idn":50, "param":"calibration_angle",              "param_idn":  17, "format":"f", "unit": "rad"         },
+  {"type_idn":50, "param":"lead_time",                      "param_idn":  18, "format":"f", "unit": "s"           },
+  {"type_idn":50, "param":"commutation_hz",                 "param_idn":  19, "format":"I", "unit": "Hz"          },
+  {"type_idn":50, "param":"phase_angle",                    "param_idn":  20, "format":"f", "unit": "rad"         },
+  {"type_idn":50, "param":"drive_volts_addition",           "param_idn":  21, "format":"f", "unit": "V"           },
+  {"type_idn":50, "param":"angle_adjust_enable",            "param_idn":  22, "format":"B", "unit": "bool"        },
+  {"type_idn":50, "param":"motor_emf_calc",                 "param_idn":  23, "format":"f", "unit": ""            },
+  {"type_idn":50, "param":"angle_adjustment",               "param_idn":  24, "format":"f", "unit": ""            },
+  {"type_idn":50, "param":"angle_adjust_max",               "param_idn":  25, "format":"f", "unit": ""            },
+  {"type_idn":50, "param":"angle_adjust_kp",                "param_idn":  26, "format":"f", "unit": ""            },
+  {"type_idn":50, "param":"angle_adjust_ki",                "param_idn":  27, "format":"f", "unit": ""            },
+  {"type_idn":50, "param":"motor_Kv",                       "param_idn":  32, "format":"f", "unit": "RPM/V"       },
+  {"type_idn":50, "param":"motor_R_ohm",                    "param_idn":  33, "format":"f", "unit": "ohm"         },
+  {"type_idn":50, "param":"motor_I_max",                    "param_idn":  34, "format":"f", "unit": "A"           },
+  {"type_idn":50, "param":"volts_limit",                    "param_idn":  35, "format":"f", "unit": "V"           },
+  {"type_idn":50, "param":"est_motor_amps",                 "param_idn":  36, "format":"f", "unit": "A"           },
+  {"type_idn":50, "param":"est_motor_torque",               "param_idn":  37, "format":"f", "unit": "Nm"          },
+  {"type_idn":50, "param":"motor_redline_start",            "param_idn":  38, "format":"f", "unit": "rad/s"       },
+  {"type_idn":50, "param":"motor_redline_end",              "param_idn":  39, "format":"f", "unit": "rad/s"       },
+  {"type_idn":50, "param":"motor_l",                        "param_idn":  40, "format":"f", "unit": "H"           },
+  {"type_idn":50, "param":"derate",                         "param_idn":  41, "format":"i", "unit": "PU fix16"    },
+  {"type_idn":50, "param":"motor_i_soft_start",             "param_idn":  42, "format":"f", "unit": ""            },
+  {"type_idn":50, "param":"motor_i_soft_end",               "param_idn":  43, "format":"f", "unit": ""            },
+  {"type_idn":50, "param":"emf",                            "param_idn":  44, "format":"f", "unit": "V"           },

```

Comment:
See my comments on this PR for details on the units that each of these should have: https://github.com/iq-motion-control/iq-module-communication-doc/pulls